com.unity.perception/Runtime/Randomization/Randomizers/RandomizerExamples/Utilities/GameObjectOneWayCache.cs

```diff
@@ -13,6 +14,8 @@ public class GameObjectOneWayCache
     {
         static ProfilerMarker s_ResetAllObjectsMarker = new ProfilerMarker("ResetAllObjects");
 
+        GameObject[] m_Prefabs;
```

Comment:
Why is this class changing?

Reply:
This is a consequence of not using archetypes to preprocess prefabs directly. In the editor, modifications made to a prefab at runtime are still permanent, so I instantiate prefabs loaded by asset sources first before I modify them. Since now you have these new archetype processed instances that need to be tracked in your project, I thought managing them using the GameObjectCache would be easier than tracking a whole separate list of spawned instances at runtime.

Partly as a consequence of this new relationship, I needed a way to use GameObject caching that duplicates sourced instances instead of sourced prefabs. The changes to the cache reflect these new requirements.